### DISCUSSION TITLE
Osu Lazer Hanging on Start screen (Arch Linux)

### DISCUSSION THREAD

**Discussion by Yqno** [ppy/osu#16685] [Q&A] [answered]
My osu lazer launcher has suddenly hung, and I don't get the welcome screen, I'm using Arch Linux, with KDE & have installed the osu-lazer-bin version from the AUR (its only work with root user)
![image](https://user-images.githubusercontent.com/79216939/151525832-b98efa12-34c5-4a94-a659-7065359008c4.png)

**Comment by peppy** [comment, marked as answer]
logs please

**Reply by smoogipoo** [reply]
Along with the logs, this is likely going to be audio related. Are you using pipewire?

**Reply by Yqno** [reply]
Yes i use Pipewire

**Reply by Yqno** [reply]
[runtime.log](https://github.com/ppy/osu/files/7958189/runtime.log)
[performance.log](https://github.com/ppy/osu/files/7958191/performance.log)
[network.log](https://github.com/ppy/osu/files/7958192/network.log)
[database.log](https://github.com/ppy/osu/files/7958193/database.log)

**Reply by bdach** [reply]
likely https://github.com/ppy/osu-framework/issues/3297

**Reply by Yqno** [reply]
@bdach thanks for the answer, I changed the audio device to AudioDevice = PulseAudio Sound Server and its works

**Comment by peppy** [comment]
Pushed a fix for this (#16710), if you're able to test that it at least gets you to the main menu in the broken scenario that would be appreciated.
